Assignements

StoicaTudor opened the pull request:
I hope you see this, please consider both of them, I commited my assignment one night before the deadline, I have the proof on my git account. I do not how git hub works and I think I haven't pull requested.

StoicaTudor committed:
Assignement 1&2

Assignement 1&2

Alex-Cosma commented on 30413/Stoica Tudor Mihai/Assignement 1/header.h:
```diff
@@ -0,0 +1,10 @@
+}node;
+
+node *first, *last;
+int nrElements,maxLengthInstruction=20;
```
this `maxLengthInstruction` should maybe be a constant => `const`

Alex-Cosma commented on 30413/Stoica Tudor Mihai/Assignement 1/main.c:
```diff
@@ -0,0 +1,343 @@
+            /// printf("%d\n",atoi(number));
+            addElementAtFront(atoi(number));
+        }
+        else
```
Gosh, this looks very ugly!
Please find a nicer way to parse the file, it's terribly simple - use `fscanf` or `strtok`

Alex-Cosma commented:
The second problem is fine

StoicaTudor committed:
Assignment 1.1&1.2

Assignment 1.1&1.2

StoicaTudor committed:
Assignement 1.1&1.2

I corrected the code in the way you said

StoicaTudor committed:
The real assignement 1.1

read this, not Assignement 1.1 simple. I don't know how to delete it from git hub

StoicaTudor committed:
The real assignement 1.1

Read this please, I do not know how to delete Assignement 1.1 from my repository

Alex-Cosma commented:
I've looked over the code - you just partially fixed the issue but parsing the file is still done in a terribly complicated manner

Alex-Cosma commented:
I encourage you to look at how your colleagues (most of them) solved the reading part - ask them, check on how they think it's best and then see what you can learn from that

StoicaTudor committed:
Assignement 2.1

Am facut frumos, cum mi-ai atras atentia la celalalt assignement :)

StoicaTudor committed:
Assignement 2.2

Done
